templates/themes/app/modules/pagerfanta/ngsite.html.twig

```diff
@@ -1,26 +1,26 @@
 {% trans_default_domain 'ngsite' %}
 
-<nav class="page-navigation">
+<nav class="page-navigation" role="navigation" aria-label="{{ 'pagination.label'|trans }}">
```

Comment:
Better translation string would be `pagination.title`.

Reply:
@emodric corrected naming